packages/dnb-eufemia/src/components/autocomplete/__tests__/Autocomplete.test.tsx

```diff
@@ -455,7 +465,7 @@ describe('Autocomplete component', () => {
     expect(Comp.find('.dnb-sr-only').first().text()).toBe('BB cc zethx')
 
     // eslint-disable-next-line
-    helpers.IS_MAC = false
+    ;(helpers.IS_MAC as any) = false
```

Comment:
@tujoworker, here as well?

Reply:
ohh, I missed this one! 

```suggestion
    Object.defineProperty(helpers, 'IS_MAC', {
      value: false,
    })
```